tests/assets/invalid_url.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
```

Comment:
Seems like this is not used at all? If you still want to write a test using this, you'll also need to tell responses to use the correct status code (404).